sdk/search/azure-search-documents/README.md

```diff
@@ -113,61 +168,258 @@ SearchAsyncClient searchAsyncClient = new SearchClientBuilder()
+            .setFilterable(true)
+            .setFacetable(true)
+            .setSortable(true)
+            .build(),
+        new SearchableFieldBuilder("postalCode", false)
+            .setFilterable(true)
+            .setFacetable(true)
+            .setSortable(true)
+            .build()
+    ))
+    .build());
+// Prepare suggester.
+SearchSuggester suggester = new SearchSuggester("sg", Collections.singletonList("hotelName"));
+// Prepare SearchIndex with index name and search fields.
+SearchIndex index = new SearchIndex("hotels").setFields(searchFieldList).setSuggesters(
+    Collections.singletonList(suggester));
+// Create an index
+searchIndexClient.createIndex(index);
+```
+
+### Retrieving a specific document from your index
+
+In addition to querying for documents using keywords and optional filters,
+you can retrieve a specific document from your index if you already know the
+key. You could get the key from a query, for example, and want to show more
+information about it or navigate your customer to that document.
+
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L213-L215 -->
+```Java 
+Hotel hotel = searchClient.getDocument("1", Hotel.class);
+System.out.println(String.format("This is hotelId %s, and this is hotel name %s.",
+    hotel.getId(), hotel.getName()));
+```
+
+### Adding documents to your index
+
+You can `Upload`, `Merge`, `MergeOrUpload`, and `Delete` multiple documents from
+an index in a single batched request.  There are
+[a few special rules for merging](https://docs.microsoft.com/rest/api/searchservice/addupdate-or-delete-documents#document-actions)
+to be aware of.
+
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L219-L222 -->
+```Java
+IndexDocumentsBatch<Hotel> batch = new IndexDocumentsBatch<Hotel>();
+batch.addUploadActions(new Hotel().setId("783").setName("Upload Inn"));
+batch.addMergeActions(new Hotel().setId("12").setName("Renovated Ranch"));
+searchClient.indexDocuments(batch);
+```
+
+The request will throw `IndexBatchException` by default if any of the individual actions fail, and you can use 
+`findFailedActionsToRetry` to retry on failed documents.
+There's also a `ThrowOnAnyError` option, and you can set it to `false` to get a successful response
+with an `IndexDocumentsResult` for inspection.
+
+
+### Async APIs
+
+All of the examples so far have been using synchronous APIs, but we provide full
+support for async APIs as well. You'll need to use [SearchAsyncClient](#Create-a-SearchClient)
+
+<!-- embedme ./src/samples/java/com/azure/search/documents/ReadmeSamples.java#L204-L209 -->
+```Java
+searchAsyncClient.search("luxury")
+    .subscribe(result -> {
+        Hotel hotel = result.getDocument(Hotel.class);
+        System.out.println(String.format("This is hotelId %s, and this is hotel name %s.",
+            hotel.getId(), hotel.getName()));
+    });
 ```
 
 - Samples are explained in detail [here][samples_readme].
```

Comment:
Should this be highlighted better?
https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/formrecognizer/azure-ai-formrecognizer#more-sample-code

Reply:
I will make it look better. 

Search has over 10 sample codes. We will take a lot of spaces to list samples codes, so we align to have link to sample Readme only across languages. 

We have a link on the top as well.
```
Source Code | API Documents | .. | Samples
```